### ISSUE TITLE
Error after running :CocInstall coc-ember 

### ISSUE THREAD

**Issue by jrowlingson** [NullVoxPopuli/coc-ember#4]
After running `:CocInstall coc-ember` the extension is installed and is listed as "active", however, completion is not working.

`:CocInfo` yields:

```
Error: Cannot find module '/Users/jack/.config/coc/extensions/node_modules/coc-ember/node_modules/@emberwatch/ember-language-server/lib/start-server.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:772:15)
    at Function.Module._load (internal/modules/cjs/loader.js:677:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:999:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
[Error  - 12:18:12 PM] Connection to server got closed. Server will not be restarted.
```

Possibly related https://github.com/NullVoxPopuli/coc-ember/commit/30743c2da8d634eaf2c9a7d680ce51d76e1de35c#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R19

^ Running the above manually (EDIT: Updated w/full error)

```
$ tsc --skipLibCheck -p .
src/definition-providers/template.ts:70:17 - error TS2742: The inferred type of 'provideRouteDefinition' cannot be named without a reference to 'coc-ember/node_modules/@emberwatch/ember-language-server/node_modules/vscode-languageserver-types'. This is likely not portable. A type annotation is necessary.

70 export function provideRouteDefinition(root: string, routeName: string) {
                   ~~~~~~~~~~~~~~~~~~~~~~


Found 1 error.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "build:language-server" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

``` v -v
NVIM v0.4.2
Build type: Release
LuaJIT 2.0.5
```

**Comment by NullVoxPopuli** [comment]
Does that type error block the rest of the transpilation/build?

**Comment by NullVoxPopuli** [comment]
Thanks for reporting this, btw!

**Comment by jrock2004** [comment]
I am getting the same issue. I was able to reproduce this by cd to the ext directory and doing a `yarn`.

![image](https://user-images.githubusercontent.com/655716/66225639-e6a45600-e6a6-11e9-9c3d-74d56f1a424d.png)

**Comment by jrock2004** [comment]
Also in nvim when I do `:CocInfo` I see

![image](https://user-images.githubusercontent.com/655716/66225894-911c7900-e6a7-11e9-8c38-f6d6409d15b1.png)

**Comment by NullVoxPopuli** [comment]
This is what I did to test:
```
# delete plugged directory (this varies per machine)
nvim (path to ember project)
:PlugInstall
re launch nvim
:CocEmber
I see the error!
```
cool, I like reproductions!

**Comment by NullVoxPopuli** [comment]
Resolved in 1.0.13
Be sure to `CocUpdate` or just nuke the coc-ember extension in `.config/coc` or wherever your extensions live.
